assets/lang/en_US.ini

```diff
@@ -435,8 +423,6 @@ This is save data, not a game. = This is save data, not a game.
 Unable to create cheat file, disk may be full = Unable to create cheat file, disk may be full.
 Unable to initialize rendering engine. = Unable to initialize rendering engine.
 Unable to write savedata, disk may be full = Unable to write savedata, disk may be full.
-Warning: Video memory FULL, reducing upscaling and switching to slow caching mode = Warning: Video memory FULL, reducing upscaling and switching to slow caching mode.
-Warning: Video memory FULL, switching to slow caching mode = Warning: Video memory FULL, switching to slow caching mode.
```

Comment:
This are still used in TextureCacheGLES:

https://github.com/hrydgard/ppsspp/blob/287dd66f20f2a03acd316a6f5d5b98ff2eb6ad26/GPU/GLES/TextureCacheGLES.cpp#L164-L166

-[Unknown]